tests/test_migrations.py

```diff
@@ -238,5 +238,67 @@ async def test_4d93784a25e5_upgrade(
         _reset_migrations()
 
 
+@pytest.mark.asyncio
+async def test_6819874e85b9_upgrade():
+    """
+    We can't create metadata by using the `client` fixture because of two reasons:
+    1) Calling the API when the db is in the downgraded state will give
```

Comment:
is #1 relevant for this migration?

Reply:
Good catch! Correct, there are not any column additions in this migration.